eng/pipelines/patch_release_client.txt

```diff
@@ -29,6 +29,7 @@ com.azure:azure-communication-identity # Tests owner: AikoBB, maximrytych-ms, mj
 com.azure:azure-communication-phonenumbers # Tests owner: whisper6284, miguhern, danielortega-msft
 com.azure:azure-communication-sms # Tests owner: besh2014, DimaKolomiiets
 com.azure:azure-communication-rooms # Tests owner: minnieliu, Mrayyan
+com.azure:azure-communication-callautomation # Tests owner: minwoolee-msft, juntuchen-msft
```

Comment:
The changes look good.

NIT: mind moving this line to before line, just wanna keep the order aligned.
`com.azure:azure-communication-chat # Tests owner: jiminwen`.

Reply:
Sounds good - i moved to match alphabetical order